docs/resources/Channel.md

```diff
@@ -438,13 +438,16 @@ The maximum request size when sending a message is 8MB.
 | content | string | the message contents (up to 2000 characters) | true |
 | nonce | snowflake | a nonce that can be used for optimistic message sending | false |
 | tts | boolean | true if this is a TTS message | false |
-| file | file contents | the contents of the file being sent | one of content, file, embeds (`multipart/form-data` only) |
+| file | file contents | the contents of the file being sent | when sending `multipart/form-data` at least one of content, file, embeds must be present. |
```

Comment:
```suggestion
| file | file contents | the contents of the file being sent | `multipart/form-data` only |
```

It's better to include this requirement somewhere else, like in the endpoint description, as this requirement also applies to a JSON payload (just with no file uploads).

Reply:
I think it makes sense to clarify endpoints in both the description and in the table to minimise the chance of confusing users?